src/algebra/group/pi.lean

```diff
@@ -38,6 +38,14 @@ instance monoid [∀ i, monoid $ f i] : monoid (Π i : I, f i) :=
 by refine_struct { one := (1 : Π i, f i), mul := (*), npow := λ n x i, npow n (x i) };
 tactic.pi_instance_derive_field
 
+-- This doesn't work as a `@[simp]` lemma as there is nothing to index on.
```

Comment:
What makes this different from `pi.add_apply`?

Reply:
Apparently nothing, I've marked it as a simp lemma now.

Reply:
Did you delete this lemma from the PR?